system/Autoloader/FileLocator.php

```diff
@@ -270,16 +267,16 @@ protected function getNamespaces()
      *
      * @return false|string The qualified name or false if the path is not found
      */
-    public function findQualifiedNameFromPath(string $path)
+    public function findQualifiedNameFromPath(string $path): false|string
```

Comment:
don't add return type on public/protected method on non-final class for framework, as it can be extended by user.